docs/extras/guides/debugging.md

```diff
@@ -8,7 +8,7 @@ Here's a few different tools and functionalities to aid in debugging.
 
 ## Tracing
 
-Platforms with tracing capabilities like [LangSmith](/docs/guides/langsmith/) and [WandB](/docs/integrations/providers/wandb_tracing) are the most comprehensive solutions for debugging. These platforms make it easy to not only log and visualize LLM apps, but also to actively debug, test and refine them.
```

Comment:
these are not typos, they are relative links

Reply:
@obi1kenobi , Yes you are correct these are the relative links, but they causing the error while clicking on these, Redirecting me to 404 Error. Let me know we need to stick to the relative links by adding any necessary files that could open by.

Thanks 😊